types/sip.js/index.d.ts

```diff
@@ -231,6 +231,16 @@ export namespace WebRTC {
     }
 }
 
+export interface TransportOptions {
+    wsServers?: string | string[];
+    connectionTimeout?: number;
+    maxReconnectionAttempts?: number;
+    reconnectionTimeout?: number;
+    keepAliveInterval?: number;
+    keepAliveDebounce?: number;
+    traceSip?: number;
```

Comment:
Seems like this is a `boolean` as shown [here](https://sipjs.com/api/0.11.0/ws_transport_configuration_parameters/#tracesip).